packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -56,12 +57,34 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
       }),
       ['radioInnerCircleSize', 'radioVisibility', 'radioFill'],
     ),
+    content: buildProps(
+      () => ({
+        style: {
+          alignItems: 'flex-start',
+          flexDirection: 'column',
+        },
+      }),
+      [],
+    ),
     label: buildProps(
-      (tokens: RadioTokens) => ({
+      (tokens: RadioTokens, theme: Theme) => ({
         variant: tokens.variant,
         style: {
           marginTop: 2,
           color: tokens.color,
+          ...fontStyles.from(tokens, theme),
+        },
+      }),
+      ['color', ...fontStyles.keys],
+    ),
+    subtext: buildProps(
+      (tokens: RadioTokens) => ({
+        variant: tokens.variant,
+        style: {
+          marginTop: globalTokens.spacing.xxs,
+          marginRight: globalTokens.spacing.xxs,
+          color: tokens.color,
+          fontSize: tokens.subtextSize,
         },
       }),
       ['color'],
```

Comment:
Cache key needs to include all tokens that would invalidate the style. In this case that would be variant, color, and subtextSize

Reply:
Fixed, thanks!